clients/include/utility.h

```diff
@@ -120,9 +120,11 @@ inline bool hipblas_isnan(float arg)
 {
     return std::isnan(arg);
 }
+
 inline bool hipblas_isnan(hipblasHalf arg)
 {
-    return (~arg & 0x7c00) == 0 && (arg & 0x3ff) != 0;
+    auto half_data = static_cast<unsigned short>(arg);
+    return (~(half_data)&0x7c00) == 0 && (half_data & 0x3ff) != 0;
 }
```

Comment:
for consistency of formatting can you add spaces on either side of & to get (~(half_data) & 0x7c00) in place of (~(half_data)&0x7c00) . This is unless clang-format removes the spaces.

Reply:
Tried changing this and clang-format seems to remove them. clang-format also adds them in the later statement when I tried to remove them.